mkosi/manifest.py

```diff
@@ -234,7 +234,21 @@ def has_data(self) -> bool:
         return len(self.packages) > 0
 
     def as_dict(self) -> Dict[str, Any]:
+        image = {
+            "distribution": self.args.distribution.name,
+            "release": self.args.release,
+            "architecture": self.args.architecture,
+            "format": self.args.output_format.name,
+        }
+        if self.args.image_id is not None:
+            image["name"] = self.args.image_id
+        else:
+            image["name"] = "image"
+        if self.args.image_version is not None:
+            image["version"] = self.args.image_version
+
         return {
+            "image": image,
```

Comment:
While at it, we could maybe add a `_manifest_version` key, since we'll probably touch this in the future and if we ever get external users this would make their life easier.

Such a key could either be the same as the mkosi release version or start at 1 now and be counted up on every release that touches the format. I'd lean towards the latter.